build_stage3-bolt.bash

```diff
@@ -9,57 +9,61 @@ cd stage3-bolt
 
 echo "== Configure Build"
 echo "== Build with stage2-prof-use-tools -- $CPATH"
+
 CC=${CPATH}/clang CXX=${CPATH}/clang++ LD=${CPATH}/lld \
-cmake 	-G Ninja \
+	cmake 	-G Ninja \
 	-DBUILD_SHARED_LIBS=OFF \
 	-DCMAKE_BUILD_TYPE=Release \
 	-DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
 	-DCLANG_ENABLE_ARCMT=OFF \
 	-DCLANG_ENABLE_STATIC_ANALYZER=OFF \
-	-DCLANG_VENDOR="LogMeIn" \
+	-DCLANG_PLUGIN_SUPPORT=OFF \
+	-DLLVM_ENABLE_BINDINGS=OFF \
+	-DLLVM_ENABLE_OCAMLDOC=OFF \
+	-DLLVM_INCLUDE_EXAMPLES=OFF \
+	-DLLVM_INCLUDE_TESTS=OFF \
+	-DLLVM_INCLUDE_DOCS=OFF \
+	-DCLANG_VENDOR="Clang-BOLT" \
 	-DLLVM_ENABLE_LLD=ON \
-	-DLLVM_ENABLE_PROJECTS="clang;lld;compiler-rt" \
+	-DCMAKE_C_FLAGS="-march=native -O3" \
+	-DCMAKE_CXX_FLAGS="-march=native -O3" \
+	-DLLVM_ENABLE_PROJECTS="clang" \
 	-DLLVM_PARALLEL_COMPILE_JOBS="$(nproc)"\
 	-DLLVM_PARALLEL_LINK_JOBS="$(nproc)" \
-	-DLLVM_POLLY_BUILD=ON \
-	-DLLVM_TARGETS_TO_BUILD="all" \
-	-DLLVM_TOOL_CLANG_BUILD=ON \
-	-DLLVM_TOOL_CLANG_TOOLS_EXTRA_BUILD=OFF \
-	-DLLVM_TOOL_COMPILER_RT_BUILD=ON \
-	-DLLVM_TOOL_LLD_BUILD=ON \
+	-DLLVM_TARGETS_TO_BUILD="X86" \
 	../../llvm-project/llvm || (echo "Could not configure project!"; exit 1)
 
 echo
 echo "== Start Training Build"
-perf record -o ../perf.data -e cycles:u -j any,u -- ninja clang || (echo "Could not build project for training!"; exit 1)
+perf record -o ../perf.data -e cycles:u -j any,u -- ninja clang  || (echo "Could not build project for training!"; exit 1)
 
 cd ..
 
 # Do the bolt-processing of the binary.
 export PATH="${BOLT_INSTALL}:$PATH"
 
 echo "* Bolting Clang"
-perf2bolt ${CPATH}/clang-7 \
+perf2bolt ${CPATH}/clang-14 \
```

Comment:
adding `bolt` to `PATH` here is not good.
this can simply be a reference to the stage1 build's binaries

Reply:
@ptr1337 could you please use `"${STAGE_ONE}/perf2bolt` here instead of the `${PATH}` change?
This is then consistent on how we use the staged compilers.